v2/internal/pkg/batch/worker.go

```diff
@@ -63,106 +68,104 @@ func (o *Batch) Worker(ctx context.Context, collectorSchema v2alpha1.CollectorSc
 	var errArray []mirrorErrorSchema
 
 	totalImages := len(collectorSchema.AllImages)
-	var countTotal, countReleaseImages, countOperatorsImages, countAdditionalImages,
-		countErrorTotal, countReleaseImagesErrorTotal, countOperatorsImagesErrorTotal, countAdditionalImagesErrorTotal int
 
 	o.Log.Info("🚀 Start " + mirrorMsg + " the images...")
 
 	for _, img := range collectorSchema.AllImages {
-		switch img.Type {
-		case v2alpha1.TypeOCPRelease, v2alpha1.TypeOCPReleaseContent, v2alpha1.TypeCincinnatiGraph:
-			countReleaseImages++
-		case v2alpha1.TypeOperatorCatalog, v2alpha1.TypeOperatorBundle, v2alpha1.TypeOperatorRelatedImage:
-			countOperatorsImages++
-		case v2alpha1.TypeGeneric:
-			countAdditionalImages++
-		}
-
-		countTotal++
-		var overalProgress string
-		if countErrorTotal > 0 {
-			overalProgress = fmt.Sprintf("=== Overall Progress - "+mirrorMsg+" image %d / %d (%d errors)===", countTotal, totalImages, countErrorTotal)
-		} else {
-			overalProgress = fmt.Sprintf("=== Overall Progress - "+mirrorMsg+" image %d / %d ===", countTotal, totalImages)
-		}
-		o.Log.Info(overalProgress)
-		if opts.Function == string(mirror.CopyMode) {
-			if countReleaseImagesErrorTotal > 0 {
-				o.Log.Info(mirrorMsg+" release image %d / %d (%d errors)", countReleaseImages, collectorSchema.TotalReleaseImages, countReleaseImagesErrorTotal)
-			} else {
-				o.Log.Info(mirrorMsg+" release image %d / %d", countReleaseImages, collectorSchema.TotalReleaseImages)
-			}
-			if countOperatorsImagesErrorTotal > 0 {
-				o.Log.Info(mirrorMsg+" operator image %d / %d (%d errors)", countOperatorsImages, collectorSchema.TotalOperatorImages, countOperatorsImagesErrorTotal)
-			} else {
-				o.Log.Info(mirrorMsg+" operator image %d / %d", countOperatorsImages, collectorSchema.TotalOperatorImages)
-			}
-			if countAdditionalImagesErrorTotal > 0 {
-				o.Log.Info(mirrorMsg+" additional image %d / %d (%d errors)", countAdditionalImages, collectorSchema.TotalAdditionalImages, countAdditionalImagesErrorTotal)
-			} else {
-				o.Log.Info(mirrorMsg+" additional image %d / %d", countAdditionalImages, collectorSchema.TotalAdditionalImages)
-			}
-			o.Log.Info(strings.Repeat("=", len(overalProgress)))
-		}
-
-		o.Log.Debug(mirrorMsg+" image: %s", img.Origin)
 
 		if img.Type == v2alpha1.TypeCincinnatiGraph && (opts.Mode == mirror.MirrorToDisk || opts.Mode == mirror.MirrorToMirror) {
 			continue
 		}
 
 		err := o.Mirror.Run(ctx, img.Source, img.Destination, mirror.Mode(opts.Function), &opts)
-
-		if err != nil {
+		isSafe := isFailSafe(err)
+		switch {
+		case err == nil:
+			o.CopiedImages.AllImages = append(o.CopiedImages.AllImages, img)
+			// switch img.Type {
```

Comment:
@sherine-k - maybe just remove the code, if not used

Reply:
Nice catch!
Even if not used by the caller right now, it can come in handy later.
Uncommented.